src/gui/dialogs/multiplayer/lobby.cpp

```diff
@@ -693,12 +694,16 @@ void mp_lobby::update_playerlist()
 		tree_group_field["use_markup"] = "true";
 		tree_group_item["name"] = tree_group_field;
 
-		tree_view_node& player = target_list->tree->add_child("player", tree_group_item);
+		//tree_view_node& player = target_list->tree->add_child("player", tree_group_item);
```

Comment:
I assume commented code will be removed before merging, right?

Reply:
Yes.